packages/@lwc/integration-types/scripts/update-paths.js

```diff
@@ -41,6 +41,8 @@ async function generatePaths(moduleDirs) {
             }
         }
     }
+    // workaround for: https://github.com/vitest-dev/vitest/issues/4567
+    paths['rollup/parseAst'] = ['../../../node_modules/rollup/dist/parseAst'];
```

Comment:
FYI @wjhsf, stole this from https://github.com/rollup/rollup/issues/5199#issuecomment-2095374821